test/Microsoft.AspNetCore.Mvc.IntegrationTests/AuthorizeFilterIntegrationTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.IntegrationTests
+{
+    public class AuthorizeFilterIntegrationTest
+    {
+        [Fact]
+        public async Task AuthorizeFilter_CalledTwiceWithNonDefaultProvider()
```

Comment:
Wouldn't hurt to add a comment about **why** this is important. For instance, what if future-ryan wants to delete this test? Why should he not?